Formula/dbt-redshift@1.1.0-rc1.rb

```diff
@@ -0,0 +1,263 @@
+class DbtRedshiftAT110Rc1 < Formula
+  include Language::Python::Virtualenv
+
+  desc "Redshift adapter plugin for dbt"
+  homepage "https://github.com/dbt-labs/dbt-redshift"
+  url "https://files.pythonhosted.org/packages/34/43/8630300360766d991738aa2ae9afc5a06c619bf451dcc1fa7c54ce85f4df/dbt-redshift-1.0.0.tar.gz"
```

Comment:
@kwigley something is wonky here where the version number is `1.0.0` when I put in `1.1.0-rc1` which is correct in other places in the file but just not right here. I'll try and look closer at this too but wasn't sure if anything popped out at you as to why this might be

Reply:
fixed here https://github.com/dbt-labs/homebrew-dbt/pull/91

Reply:
we should be able to just re-run the action after https://github.com/dbt-labs/homebrew-dbt/pull/91 is merged